frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -139,7 +142,10 @@ function HeadingWithAnchor({
         return
       }
 
-      const anchor = propsAnchor || createAnchorFromText(node.textContent)
+      const anchor =
+        propsAnchor ||
+        createAnchorFromText(node.textContent) ||
+        "unknown-anchor"
```

Comment:
The edge cases here are tricky -- if multiple calls to `st.markdown()` end up getting assigned the anchor `"unknown-anchor"`, the anchor links all end up pointing to the first one, which doesn't seem great. I also think having auto inserted `"unknown-anchor"` anchors might be more confusing than just not having any at all

I wonder if we should instead just not try to auto-insert an anchor at all in this case, or maybe we could hash `node.textContent` and use that as the anchor? This might need some product input :neutral_face:

Reply:
Good point. I think you're right, probably better to just not show an anchor. I removed this change.